packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -231,14 +229,10 @@ contract Bridge is EssentialContract, IBridge, BridgeErrors {
      * Check if the Ether associated with the given message hash has been
      * released.
      * @param msgHash The hash of the message.
-     * @return Returns true if the Ether has been released, false otherwise.
```

Comment:
recall message may release Ether, but please find and remove all the "release Ether" doc-comments, instead, let the IRecallableMessageSender to decide whether the Ether should be returned to message.owner or not.